x/claims/keeper/integration_test.go

```diff
@@ -432,6 +432,9 @@ func deliverTx(priv *ethsecp256k1.PrivKey, msgs ...sdk.Msg) abci.ResponseDeliver
 	txBuilder := encodingConfig.TxConfig.NewTxBuilder()
 
 	txBuilder.SetGasLimit(1000000)
+	amt, _ := sdk.NewIntFromString("1000000000000000")
+	txBuilder.SetFeeAmount(sdk.Coins{{Denom: evm.DefaultEVMDenom, Amount: amt}})
```

Comment:
this should be `BaseDenom`

Reply:
I tried this but changes some test results. Will make the corresponding refactor